assets/src/blocks/Articles/ArticlesBlock.js

```diff
@@ -36,19 +36,15 @@ export class ArticlesBlock {
           type: 'array',
           default: []
         },
-        post_types: {
```

Comment:
I don't think we're supposed to remove the `post_types` filter. [The ticket](https://jira.greenpeace.org/browse/PLANET-6519) only mentions

> Remove the post type and tags from the block.

Reply:
Yes that is something that was agreed upon on Slack with Mag directly, but as we discussed it's not a good idea to remove it right now without checking existing blocks, it requires more thinking through and therefore [I've added it back](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/808/commits/1101b43aa4ec1fc8bc4dd34618a3cd3e470211d3).